### PR TITLE
Adjusted Grafana configs to allow multiple providers for a dashboard.

### PR DESCRIPTION
**Pull request by hjdjoo**
Saw that we had several inputs for our Grafana, but this doesn't seem like something that's necessary unless we want these inputs to be provided by the user when they are importing our instance to a new Grafana instance (or something like that).

I saw that we were already implementing templating to be able to select a "Prometheus" provider from the dashboard. Turns out, we just need a single provider in that case, and it doesn't actually need to be named "Prometheus" anything - just something descriptive. The only thing that needed to happen was for the new Prometheus instance to be added as a datasource.

WHEN a user creates a new Prometheus instance by submitting a Connections form,
THEN the dashboard.yml and .json files are unchanged,
AND the new prometheus instance is added to the list of datasources.
THEN the user can visit localhost:3000 if desired and select from the various instances of Prometheus as the datasource for a particular panel.

### CODE REVIEW

**Review by echochs1 — COMMENTED** [review]
Merge to dev branch not main

**Comment by hjdjoo** [comment]
> Merge to dev branch not main

What kind of monster would merge directly into main

File: .ssl_certificates/localhost+2-key.pem
Comment (echochs1) [inline .ssl_certificates/localhost+2-key.pem:1]: delete

File: .ssl_certificates/localhost+2.pem
Comment (echochs1) [inline .ssl_certificates/localhost+2.pem:1]: delete

**Review by echochs1 — APPROVED** [review]

**Pull request merged**